sc_wgs_monitoring/check.py

```
@@ -0,0 +1,64 @@
+        String to check
+
+    Returns
+    -------
+    bool
+        Result of the regex
+    """
+
+    return (
+        True
+        if re.search(r"file|project|job-[0-9a-zA-Z]{24}", string)
+        else False
```

Ruff (0.11.9)

Use `bool(...)` instead of `True if ... else False`

Replace with `bool(...)

(SIM210)